README.md

```diff
@@ -15,7 +15,7 @@ Note: for OS X and Unix users, [Mono 4.2 or higher](http://www.mono-project.com/
 
 ## Contribution Guidelines
 
-The `dom.generated.d.ts` and `webworker.generated.d.ts` files in the TypeScript repo are used as baselines.
+The `dom.generated.d.ts`, `webworker.generated.d.ts` an`dom.es6.generated.d.ts` files from the TypeScript repo are used as baselines.
```

Comment:
Typo: an -> and

Reply:
I'll fix this when I have the final answer in the issue, thanks.